react/features/base/participants/subscriber.js

```diff
@@ -47,12 +47,15 @@ function _updateScreenshareParticipants({ getState, dispatch }) {
         return acc;
     }, []);
 
-    if (!localScreenShare && newLocalSceenshareSourceName) {
-        dispatch(createVirtualScreenshareParticipant(newLocalSceenshareSourceName, true));
-    }
+    // Mobile does not yet create a new video stream, it replaces the camera with a desktop stream.
+    if (navigator.product !== 'ReactNative') {
```

Comment:
If we plan to never send multiple stream from RN or not automatically switch to this mode when RN switches to Unified plan, I would recommend adding !(RN) check to https://github.com/jitsi/jitsi-meet/blob/master/react/features/base/config/functions.any.js#L70 and use getMultipleVideoSendingSupportFeatureFlag check here. That would also fix cases where the code would automatically assume multiple stream send support when RN switches to Unified plan.

Reply:
Oh that's a good idea indeed! I'll make the change!